owl/EASE-ACT.owl

```diff
@@ -416,22 +490,42 @@ In this category we will place Actions for which such extra knowledge is require
     <!-- http://www.ease-crc.org/ont/EASE-ACT.owl#Leaning -->
 
     <owl:Class rdf:about="http://www.ease-crc.org/ont/EASE-ACT.owl#Leaning">
-        <rdfs:subClassOf rdf:resource="http://www.ease-crc.org/ont/EASE-ACT.owl#PhysicalTask"/>
+        <rdfs:subClassOf rdf:resource="http://www.ease-crc.org/ont/EASE-ACT.owl#PosturalMoving"/>
+        <ease:ELANName>move-body-lean</ease:ELANName>
+        <ease:ELANUsageGuideline>move, bending, with torso in a specific direction</ease:ELANUsageGuideline>
     </owl:Class>
     
 
 
     <!-- http://www.ease-crc.org/ont/EASE-ACT.owl#Lifting -->
 
     <owl:Class rdf:about="http://www.ease-crc.org/ont/EASE-ACT.owl#Lifting">
-        <rdfs:subClassOf rdf:resource="http://www.ease-crc.org/ont/EASE-ACT.owl#PhysicalTask"/>
+        <rdfs:subClassOf rdf:resource="http://www.ease-crc.org/ont/EASE-ACT.owl#Actuating"/>
         <owl:disjointWith rdf:resource="http://www.ease-crc.org/ont/EASE-ACT.owl#Opening"/>
         <owl:disjointWith rdf:resource="http://www.ease-crc.org/ont/EASE-ACT.owl#Pushing"/>
+        <ease:ELANName>move-object-lift</ease:ELANName>
+        <ease:ELANUsageGuideline>Having grasped an object with their hand, the agent now lifts it from the supporting surface.</ease:ELANUsageGuideline>
         <rdfs:comment>todo: how to distinguish from e.g. &apos;pushing from the table&apos;</rdfs:comment>
     </owl:Class>
     
 
 
+    <!-- http://www.ease-crc.org/ont/EASE-ACT.owl#Locomotion -->
+
+    <owl:Class rdf:about="http://www.ease-crc.org/ont/EASE-ACT.owl#Locomotion">
+        <rdfs:subClassOf rdf:resource="http://www.ease-crc.org/ont/EASE-ACT.owl#PhysicalTask"/>
+        <ease:ELANName>move-body-general</ease:ELANName>
+        <ease:ELANUsageGuideline>Superconcept for the Agent moving around; use more specific labels where appropriate.</ease:ELANUsageGuideline>
+        <rdfs:comment>Conceptually overlaps with Navigation, but distinguishable from it because of the profile, ie. the focus of the task.
+
+Navigation is about reaching some goal.
+
+Locomotion is concerned more with the actual motion.</rdfs:comment>
```

Comment:
*Locomotion* would not be a task, but a *MotionType*, or?

Well you could tell someone to "move more" when this person is in front of the computer the whole day for example ;) But then the task would be rather *Exercise* or so, I think.

Reply:
Same as headmovement, I'll put this under motion type.

Reply:
Now placed under body movement.

Reply:
Also placed PosturalMoving under the BodyMovement class.

As a result, deleted an axiom from PosturalMoving that it only classifies PosturalAction. At some point, something like this axiom should be reinstated, but we need to clarify whether PosturalAction isn't better regarded as a Process then.